### DISCUSSION TITLE
Temp TMEAT freeze unclear/incorrect

### DISCUSSION THREAD

**Discussion by DJshelfmushroom** [3kh0/ext-remover#916] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
step 6 says repeat step 3. why? that does not make sense. does it mean step 5?

**Comment by AshtonDavies** [comment]
What kind of circumstances have you using Temp Tmeat anyway? No one really uses it.

**Reply by DJshelfmushroom** [reply]
mostly LTMEAT original blocked

**Reply by CGRmags** [reply]
use LMEET flood?

**Reply by DJshelfmushroom** [reply]
seemed like a lot of work idk

**Reply by CGRmags** [reply]
just do it its not THAT hard but it takes like 7-10 minutes

**Reply by DJshelfmushroom** [reply]
yeeeah I can if all the others fail.

**Reply by CGRmags** [reply]
ok do whatever you want if you have more question just ask

**Reply by DJshelfmushroom** [reply]
yeah the actual issue?

**Reply by AshtonDavies** [reply]
Visit [here](https://chrose.netlify.app/detail/ltmeat%20flood) for the official page and proper setup instructions. It calculates your folder capacity for you.

**Reply by DJshelfmushroom** [reply]
thanks! (ugh 2000)

**Reply by DJshelfmushroom** [reply]
wait there's no "allow access to file urls" switch

**Reply by AshtonDavies** [reply]
> wait there's no "allow access to file urls" switch

That's a common issue for some extraordinary filtering extensions, and it supposedly means that your extension doesn't request the browser to wait for a response before proceeding to load pages.

If that's the case, you can skip that step.
